langkit/input_output.py

```diff
@@ -21,10 +21,9 @@ def init(transformer_name: Optional[str] = None):
 
 
 init()
-_udf_name = _transformer_name or lang_config.transformer_name
 
 
-@register_dataset_udf(["prompt", "response"], f"similarity_{_udf_name.split('/')[-1]}")
+@register_dataset_udf(["prompt", "response"], "response.relevance_to_prompt")
```

Comment:
Looks like better naming per offline discussion.

There are --load tests that aren't run in the CI yet when the models are larger/heavier to load, that assumes the old name, so we'll need to update the unit tests.

Reply:
Tests updated in https://github.com/whylabs/langkit/pull/36/commits/158a22a52b7111c7f992a8eb2a6848d5b839a38e